.env

```diff
@@ -0,0 +1 @@
+ATLAS_URI=mongodb+srv://jakelop:soen490@cluster0.nsqs3nv.mongodb.net/?retryWrites=true&w=majority
```

Comment:
Do not commit this file. Please add it to the gitignore and share it on discord. This repo is public.